src/client/array/dc_array.c

```diff
@@ -1708,8 +1711,8 @@ dc_array_io(daos_handle_t array_oh, daos_handle_t th,
 					      tse_task2sched(task),
 					      0, NULL, &io_task);
 			if (rc != 0) {
-				D_ERROR("Fetch dkey "DF_U64" failed (%d)\n",
-					params->dkey_val, rc);
+				D_ERROR("Fetch dkey " DF_U64 " failed " DF_RC "\n",
```

Comment:
I think better to fix the checkpatch script as "DF_U64" such usage is reasonable.

Reply:
We've got code which uses both styles, but the majority of this file seems to be the shorted one so I've updated it.

This came up in the chat recently, we should probably decide to standardise on no spaces and update checkpatch as you say.